src/main/resources/application.conf

```diff
@@ -1 +1 @@
-pandoc.compile.command="pandoc"
\ No newline at end of file
+pandoc.compile.command="launch-pandoc %s %s %s %s %s"
```

Comment:
I understand that it is the same as it was in the legacy code, but now its time to make it better. Becuase it is good when you understand what is supposed to be the first argument, the second, etc., rather than guessing. Can you please these "positional" placeholders with named,  like this:
```
launch-pandoc ${rootFilePath} ${rootFileName} ${outputFile}
```
There is a nice [StringSubstitutor](https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/StringSubstitutor.html) from commons-text which can do that. Notice that you'll have to escape `$` in config because typesafe config also treats `${}` as substitution

Reply:
Here's what I don't really like: I have to put same values into the list in kotlin to substitute them.
looks like a duplicate: if we change paramater name in `.conf` file, we need also change same parameter in list.